lib/metadata/bucketfile/backend.js

```diff
@@ -8,7 +8,7 @@ import constants from '../../../constants';
 import config from '../../Config';
 
 const errors = arsenal.errors;
-const MetadataClient = arsenal.storage.metadata.client;
+const MetadataClient = arsenal.storage.metadata.MetadataFileClient;
```

Comment:
This is because this PR has a dependency on https://github.com/scality/S3/pull/686, so it actually contains the changes.